src/nightly_boards/n87.json

```diff
@@ -0,0 +1,294 @@
+{
+  "name": "Nightly Boards [n]87",
+  "vendorId": "0x0717",
+  "productId": "0x0001",
+  "lighting": "qmk_rgblight",
+  "matrix": {
+    "rows": 12,
+    "cols": 9
+  },
+  "layouts": {
+    "labels": [
+      "Split Backspace",
+      "Split Right Shift",
+      ["Bottom Row","6.25U","Centered 6.25U","Tsangan"]
```

Comment:
We prefer "7U" instead of "Tsangan".

Reply:
Okay thanks! Will edit it

Reply:
https://github.com/the-via/keyboards/pull/241/commits/b8fbb9e3bbed58867221cfaca3e93fab28ac0fa6